### PR TITLE
Fix responsive images configuration for recipes

### PR DESCRIPTION
**Pull request by tatitati**
Added different "sizes" for recipes as the panel and the full page are displayed at different sizes.

**Urls to test it:**
Full page: http://sandbox.bbc.co.uk/programmes/b013pqnm/recipes?__flush_cache=1
Episode page with recipes panel: https://sandbox.bbc.co.uk/programmes/b0876646?__flush_cache=1

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Presenters/Domain/Recipe/recipe.html.twig [hunk 1]
```diff
@@ -5,8 +5,7 @@
                 ds_shared('image',
                     recipe.getImage(),
                     280,
-                    { 0: 1/2, 768: 1/4, 1008: '237px' },
-                    {
+                    recipe.getOption('srcset'),{
```
Comment (benolot) [inline src/Ds2013/Presenters/Domain/Recipe/recipe.html.twig]: Should the array not start on a new line?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by tatitati** [comment]
I will open a new PR to make clearer the twig template names

**Pull request merged**